tests/test_database_interface.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import sys
+import json
+settings_file = open('.\\settings.json')
+settings = json.load(settings_file)
+sys.path.append(settings["local path to PowerFactory application"])
+import powerfactory
+sys.path.insert(0, r'.\src')
+import powfacpy 
+import importlib
+importlib.reload(powfacpy)
+from jsondiff import diff
+
+from test_base_interface import pfbi, pf_app, activate_test_project
+
+@pytest.fixture
+def pfdbi(pf_app):
+  # Return PFDataBaseInterface instance
+  return powfacpy.PFDatabaseInterface(pf_app)
+
+def test_get_and_set_object_attributes(pfdbi, activate_test_project):
+  """
+  Tests for getting and setting object attributes.
+  The methods get_object_attributes and set_object_attributes can best be tested
+  together by 
+    1. getting data of PF objects from the PF database
+    2. setting the data of the PF objects in the PF database with the output from 1.
+    3. again getting data of PF objects from the PF database
+    4. Checking if the output from 1. and 3. are equal. If so, gettind and setting
+    data works correctly.
```

Comment:
ok, now the data (i.e. the dict that was read in step 1) is change between step 1 and 2 and then also used for comparison in step 4. (Random values should be avoided in unit testing, because the test will not be reproducible)

Reply:
agreed. 
The way I intended it was to use a fixed random seed (np.random.seed(1234) ) to get reproducible "random" values.